spec/fortnox/api/models/base_spec.rb

```diff
@@ -1,52 +1,75 @@
 require 'spec_helper'
 require 'fortnox/api/models/base'
 
-class TestEntity < Fortnox::API::Model::Base
-  attribute :private, String, writer: :private
-  attribute :string, String
-  attribute :number, Integer, default: 42
-end
-
 describe Fortnox::API::Model::Base do
-  let(:original){ TestEntity.new( string: 'Test' ) }
+  using_test_classes do
+    class TestEntity < Fortnox::API::Model::Base
```

Comment:
I really do not like prefixes like this :) It's obviously a test entity, it's in a test (and we don't have any other model named `Entity` in our gem).